clients/src/main/resources/common/message/FetchRequest.json

```diff
@@ -84,6 +84,8 @@
         "about": "The partition name." },
       { "name": "ForgottenPartitionIndexes", "type": "[]int32", "versions": "7+",
         "about": "The partitions indexes to forget." }
-    ]}
+    ]},
+    { "name": "RackId", "type":  "string", "versions": "11+", "default": "", "ignorable": true,
```

Comment:
Do we want to let this be nullable? Looks like we are using null in a few test cases below.

Reply:
Discussed separately. We'll leave this non-nullable with "" as the default.